centaur/test_cromwell.sh

```diff
@@ -24,8 +24,7 @@ Arguments:
     -r    Directory where script is run (defaults to current directory)
     -g    Generate code coverage output for the centaur main classes
     -c    If supplied, the config file to pass to Cromwell
-    -t    If supplied, the file containing a Refresh Token that can be passed into the appropriate options file
-    -s    If supplied, the Google service account json that should be used for various centaur operations
+    -t    If supplied, the timeout for request-plus-response from Centaur to Cromwell
```

Comment:
those really weren't being used?

Reply:
They were removed when JES moved its centaur creds from GCS to vault. The docs just weren't updated at the time.